react/components/EXPERIMENTAL_FilterBar/index.js

```diff
@@ -0,0 +1,235 @@
+/* eslint-disable camelcase */
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import ButtonWithIcon from '../ButtonWithIcon'
+import IconClose from '../icon/Close'
+
+import FilterTag from './FilterTag'
+
+const HEAVY_ICON_OPTICAL_COMPENSATION = { marginTop: '1px' }
+
+const isStatementComplete = st => st.subject && st.verb && st.object
+const filterExtraOptions = (options, alwaysVisibleFilters, statements) => {
+  const newOptions = { ...options }
+  const optionsKeys = Object.keys(options)
+  optionsKeys.forEach(key => {
+    if (alwaysVisibleFilters.includes(key)) {
+      delete newOptions[key]
+    }
+  })
+  statements.forEach(st => {
+    if (st && st.object) {
+      delete newOptions[st.subject]
```

Comment:
:drake-no:

Reply:
You can just do a map taking out this key or using `omit` from ramda